doc/radosgw/upgrade_to_jewel.rst

```diff
@@ -0,0 +1,37 @@
+=============
+RGW upgrade procedure to older version of jewel
```

Comment:
ah.. now I understand.. maybe: `upgrading to Jewel versions 10.2.0, 10.2.1, 10.2.2`